packages/components/Button/src/Button.types.ts

```diff
@@ -5,7 +5,7 @@ import { ITextProps } from '@fluentui-react-native/text';
 import { IPressableProps } from '@fluentui-react-native/pressable';
 import { FontTokens, IForegroundColorTokens, IBackgroundColorTokens, IBorderTokens } from '@fluentui-react-native/tokens';
 import { IFocusable, IPressableState } from '@fluentui-react-native/interactive-hooks';
-import { IViewWin32Props } from '@office-iss/react-native-win32';
+import type { IViewProps } from '@fluentui-react-native/adapters';
```

Comment:
Thank you for this :D No reason the JS components should pull react-native-win32 on all platforms